src/plugin/tasks/package/snippets/serverless-dynamodb/merge-shallow.json

```diff
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "ddba": "AWS_REGION=us-west-2 AWS_ACCESS_KEY_ID=XXXXXXXXXXXXXXXXXXXX AWS_SECRET_ACCESS_KEY=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX DYNAMO_ENDPOINT=http://0.0.0.0:8000 dynamodb-admin"
```

Comment:
the region should come from the var (everywhere us-west-2 is used!)

Reply:
Later we can check for this other ones if needed.
https://github.com/blackflux/robo-config-plugin/blob/565aaa5e422769d808b2ac868a0f9c8645e9f717/test/projects/assorted/%40sls-closedsource-dynamodb/test/0-sls-cf-stack.spec.js#L19
Because will be in the same path of tests but it is not related since it is only checking for configuration.